src/main/webapp/stylesheets/teammatesCommon.css

```diff
@@ -304,6 +304,10 @@ Apply to thead to make headers not bold.
     }
 }
 
+.table-column-checkbox {
+    float: none !important;
+}
+
```

Comment:
Got it.
May be good to mention its no float property in the class's name somehow, though.